webui/react/src/e2e/models/components/NavigationSideBar.ts

```diff
@@ -52,21 +52,27 @@ export class NavigationSideBar extends BaseReactFragment {
     parent: this.#nav,
     selector: `a[aria-label="${'Workspaces'}"]`,
   });
-  readonly createWorkspace = new BaseComponent({
+  readonly createWorkspaceFromHover = new BaseComponent({
     parent: this.#nav,
     selector: 'span[aria-label="Create workspace"]',
   });
+  // consider this to be a rowContainer
+  readonly #pinnedWorkspaces = new BaseComponent({
```

Comment:
there's a few elements that get grouped inside this component, like the second create workspace button. this way, I can be more precise in `sidebarWorkspaceItem`'s parent. Since we're not clicking or interacting with the container, I chose to make it private.